api/groups/transactionGroup.go

```diff
@@ -465,6 +468,20 @@ func (tg *transactionGroup) sendMultipleTransactions(c *gin.Context) {
 	)
 }
 
+func adjustTxValidatorErrorIfNeeded(err error, address string) error {
+	if err == nil {
+		return nil
+	}
+
+	// TODO: might move these errors from the process package so they can be used here as well
+
+	if !strings.Contains(err.Error(), "account not found") {
+		return err
+	}
+
+	return fmt.Errorf("insufficient balance for address %s", address)
```

Comment:
this is a bad assumption that any other error other than account not found is caused by insufficient balance. I would add something like:
```
return fmt.Errorf("%w that might be caused by insufficient balance for address %s", err, address)
```

Reply:
refactored